CONTRIBUTING.md

```diff
@@ -63,29 +73,29 @@ Weaver.
     
    TODO: should we avoid discouraging multi-commit PRs?
 
-4. Pull any changes that may have been made in the shared repository
+6. Pull any changes that may have been made in the upstream repository
```

Comment:
`upstream`

Reply:
I meant to say put the word "upstream" in markdown quotes.

Reply:
Done.